crates/sui-indexer/migrations/2022-11-18-195259_transactions/up.sql

```diff
@@ -1,7 +1,15 @@
+DO $$
+BEGIN
 -- SuiAddress and ObjectId type, 0x + 64 chars hex string
 CREATE DOMAIN address VARCHAR(66);
 -- Max char length for base58 encoded digest
 CREATE DOMAIN base58digest VARCHAR(44);
+EXCEPTION
```

Comment:
cc @patrickkuo @gegaowp made this change to skip create if domain exists

Reply:
I saw this problem upon local. testing as well, thanks for putting this!

Reply:
this means the down script is not right, this is just masking the problem....
I will have a look why this is not dropped correctly in down.sql